### DISCUSSION TITLE
gpu optimus?

### DISCUSSION THREAD

**Discussion by ghost** [seerge/g-helper#1566] [General]
When is Optimus for gpu going to be added?

**Comment by seerge** [comment]
Optimus is a nvidia driver feature, you should use nvidia control panel for that

**Reply by ghost** [reply]
Not true. Cause my nvidia  control panel doesnt have optimus feature. It was in Armoury crate only. Yes my drivers are updated

**Reply by seerge** [reply]
AC doesn’t control optimus anyhow

**Reply by ghost** [reply]
In my armoury crate it will show single zone/ multizone or optimus under gpu column.  I dont get the option in nvidia control panel. Like I've seen in youtube videos.

**Reply by seerge** [reply]
I would like to see a screenshot of what you have in AC. So far you have provided 0 useful information about your device and your case.

**Reply by ghost** [reply]
I will Okay

**Reply by ghost** [reply]
Here is proof.

- No Optimus option in NVidia settings.
-  Only way to get Optimus working is through armoury crate. By clicking the multizone button and switching to Optimus. 

<img width="1267" alt="Screenshot 2023-11-12 183058" src="https://github.com/seerge/g-helper/assets/149981353/8b351bf8-8684-4e42-8689-9c32dde83ffb">
